### ISSUE TITLE
Add a testing hook for the Flash object when compiled in debug mode

### ISSUE THREAD

**Issue by JamesMGreene** [zeroclipboard/zeroclipboard#43]
In order to test the Flash object from browser-based JavaScript unit tests (e.g. QUnit, etc.), we would need to add a hook method that can be directly invoked from the JavaScript side as we cannot simulate a click event on the Flash object (unless you used Selenium, perhaps, but that's a lot of overhead).

The Flash object should not be able to actually inject into the clipboard this way but it can at least still fire the "complete" event and send the text that it would've inserted back to JavaScript land for verification.

Example, basically adds another callback via ExternalInterface:
     http://stackoverflow.com/questions/12129510/invoke-flash-click-event-from-javascript

And then we'd want to wrap that so that it is only produced when the SWF is compiled in debug mode.

**Comment by JamesMGreene** [comment]
Found one more random resource that may or may not be feasible called [FlexPilot](https://github.com/mde/flex-pilot).  It was formerly available as a [testing platform/experiment through SauceLabs](http://sauceio.com/index.php/tag/flash-test/), which a SDTimes news article and quote from Sauce JS Architect @admc claimed could fire native events to the Flash/Flex applications.  I'm not certain if that statement is 100% accurate but I thought I'd at least note it here.

**Comment by JamesMGreene** [comment]
@admc @mde: Could one of you save me a bunch of research and setup time and inform me whether FlexPilot is actually capable of sending **native** user-level `click` events to Flex applications in the browser vs. just exposing a way to invoke the underlying Flash/Flex click handling callbacks?

**Comment by mde** [comment]
IIRC it was all based on AS3's DOM event API, so probably not native events.

**Comment by JamesMGreene** [comment]
@mde Yeah, that was the impression I was getting while looking at the API and examples.
